NET/ReadyToUseUI.Droid/Model/Filter.cs

```diff
@@ -0,0 +1,39 @@
+using IO.Scanbot.Sdk.Process;
+
+namespace ReadyToUseUI.Droid.Model;
+
+public enum FilterType
```

Comment:
@scan-mayank, following my other suggestion of adding a delegate to create the filter, we can eliminate this enum and also extra properties in FilterItem (like FilterType and LegacyFilterCode)